src/processing/DataManipulator.py

```diff
@@ -0,0 +1,109 @@
+import numpy as np
+import pandas as pd
+import seaborn as sns
+from tqdm import tqdm
+import matplotlib.pyplot as plt
+from sklearn.feature_selection import chi2
+from sklearn.model_selection import cross_val_score, train_test_split
+from sklearn.feature_extraction.text import TfidfVectorizer, CountVectorizer, TfidfTransformer
+
+
+def breakdown_cat_distribution(df:pd.DataFrame, cat_columns:list):
```

Comment:
It's nice to also annotate the return type. Eg: `def breakdown_cat_distribution(df:pd.DataFrame, cat_columns:list) -> None:` as this function returns nothing